src/meson.build

```diff
@@ -32,11 +36,22 @@ configure_file(
 raqm_headers = files('raqm.h')
 install_headers(raqm_headers)
 
+# ABI compatibility version should be that used for last non-meson release, 0.7.1
+raqm_ver_compat = 701
+raqm_ver_current = raqm_ver_major * 10000 + raqm_ver_minor * 100 + raqm_ver_micro
```

Comment:
I don’t understand the significance of the last non-meson release here. All releases should be API and ABI-compatible.

Reply:
So `raqm_ver_compat` should be at 0?

Reply:
> So `raqm_ver_compat` should be at 0?

No, that only works if older versions used zero, but they didn't.

Reply:
> I really still don’t understand any of this. What part of autotools build was setting this dylib versioning? We should replicate the same behavior, not hard-code a specific version.

We'd have to go back and look at the precise behavior of the autosools build to know for sure. But what I _can_ say, is that one of the last releases set the version to that value.

So I agree, some research is needed.